posts/2021-04-16-mp-context-propagration-21004.adoc

```diff
@@ -0,0 +1,283 @@
+---
+layout: post
+title: "Admin Center, MicroProfile Context Propagration 1.2, and a number of enhancements in Open Liberty 21.0.0.4"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/jakub-pomykala
+author_github: https://github.com/jakub-pomykala
+seo-title: Admin Center, MicroProfile Context Propagration 1.2 and a number of enhancements in Open Liberty 21.0.0.4 - OpenLiberty.io
+seo-description: In 21.0.0.4, we now offer Admin Center, MicroProfile Context Propagration 1.2, and a number of enhancements (expandLocation property for AutoExpand, Ability to control whether thread dumps are collected when a hung request is detected in the Request Timing feature, Automatic Cleanup of Leaked Connections, Single Sign-On (SSO) LTPA and JWT support authentication filter and Single Sign-On (SSO) LTPA and JWT support authentication filter)
+blog_description: In 21.0.0.4, we now offer Admin Center, MicroProfile Context Propagration 1.2, and a number of enhancements (expandLocation property for AutoExpand, Ability to control whether thread dumps are collected when a hung request is detected in the Request Timing feature, Automatic Cleanup of Leaked Connections, Single Sign-On (SSO) LTPA and JWT support authentication filter and Single Sign-On (SSO) LTPA and JWT support authentication filter)
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= Admin Center, MicroProfile Context Propagration 1.2, and a number of enhancements in Open Liberty 21.0.0.4
+Jakub Pomykala <https://github.com/jakub-pomykala>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+
+// tag::intro[]
+
+In 21.0.0.4, Admin Center 1.0 joins Open Liberty, providing a web-based UI for configuring, managing, and monitoring your server, Java Batch jobs, and OpenID Connect (OIDC). MicroProfile Context Propagation is updated to 1.2, which addresses the difference in how link:{url-prefix}/blog/2021/03/31/microprofile-config-2.0.html[MicroProfile Config 2.0] treats empty value configuration properties. Also present in this release is a number of enhancements: expandLocation property for AutoExpand, ability to control whether thread dumps are collected when a hung request is detected in the Request Timing feature, automatic cleanup of leaked connections, Single Sign-On (SSO) LTPA and JWT support authentication filter, and Single Sign-On (SSO) LTPA and JWT support authentication filter.
```

Comment:
`providing a web-based UI for configuring, managing, and monitoring your server, Java Batch jobs, and OpenID Connect (OIDC).` read strangely for me the first time round, the way it's written in the `admincenter` section is much nicer imo.

Reply:
Changed to description from admin center section